### PR TITLE
🐞♻️ fixup doc issues exposed in PR9194

### PR DESCRIPTION
**Pull request by andrewconnell**
## Category

- [x] Content fix
- [ ] New article

## Related issues

- ref #9194

## What's in this Pull Request?

- fixup link issues across multiple docs exposed in previously auto-submitted PR

### CODE REVIEW

File: docs/sp-add-ins-modernize/from-acs-to-aad-apps.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ If you prefer, you can watch the following video, instead of reading the whole a
...
 
 ![The Application Registration page with form fields to register an application in ACS. The fields are: Client Id, Client Secret, Title, App Domain, and Redirect URI.](../images/add-in-transform/from-acs-to-aad-apps/from-acs-to-aad-apps-acs-appregnew.png)
 
-The application was granted *FullControl* right on the target SharePoint Online tenant through the following policy, which was registered via the https://[your-tenant]-admin.sharepoint.com/_layouts/15/appinv.aspx URL of the SharePoint Online Admin Central.
+The application was granted *FullControl* right on the target SharePoint Online tenant through the following policy, which was registered via the `https://[your-tenant]-admin.sharepoint.com/_layouts/15/appinv.aspx` URL of the SharePoint Online Admin Central.
 
 ```XML
 <AppPermissionRequests AllowAppOnlyPolicy="true">
-  <AppPermissionRequest Scope="http://sharepoint/content/tenant" Right="FullControl" />
+  <AppPermissionRequest Scope="https://sharepoint/content/tenant" Right="FullControl" />
```
Comment (ltdu) [inline docs/sp-add-ins-modernize/from-acs-to-aad-apps.md:43]: Has Microsoft really updated permission scope to have ```https```?

File: docs/sp-add-ins-modernize/from-app-parts-to-modern-web-parts.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ The App Part is then defined in an XML manifest file, where it's also defined th
 
 ```xml
 <?xml version="1.0" encoding="utf-8"?>
-<Elements xmlns="http://schemas.microsoft.com/sharepoint/">
+<Elements xmlns="https://schemas.microsoft.com/sharepoint/">
```
Comment (ltdu) [inline docs/sp-add-ins-modernize/from-app-parts-to-modern-web-parts.md:119]: ... and for XML namespace?

File: docs/sp-add-ins-modernize/from-app-parts-to-modern-web-parts.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ The App Part is then defined in an XML manifest file, where it's also defined th
     <!-- Define properties in the Properties element.
          Remember to put Property Name on the Src attribute of the Content element above. -->
     <Properties>
-      <Property xmlns="http://schemas.microsoft.com/sharepoint/"
+      <Property xmlns="https://schemas.microsoft.com/sharepoint/"
```
Comment (ltdu) [inline docs/sp-add-ins-modernize/from-app-parts-to-modern-web-parts.md:130]: ... and here as well?

**Comment by andrewconnell** [comment]
🤦‍♂️ thanks for calling that out @ltdu ... I blindly made the update based on the build check warnings. I'll revert these changes...

**Pull request merged**